b2share/modules/schemas/views.py

```diff
@@ -139,12 +147,79 @@ def get(self, community_schema):
         return community_schema
 
 
+class BlockSchemaResource(ContentNegotiatedMethodView):
```

Comment:
This needs to be split into two resource classes. BlockSchemaResource and BlockSchemaListResource